lib/stack_master/stack.rb

```diff
@@ -75,6 +75,25 @@ def self.generate(stack_definition, config)
           stack_policy_body: stack_policy_body)
     end
 
+    def self.generate_without_parameters(stack_definition, config)
```

Comment:
Given compile time parameters are resolved, consider renaming the method to be slightly more accurate. Perhaps something like `generate_without_template_parameters`?

Reply:
I'm in two minds about this. The `Stack` instance this generates has a `parameters` property which only includes the template parameters. Compile-time parameters don't seem to be a thing once the `Stack` instance has been generated.

Reply:
> I'm in two minds about this. The Stack instance this generates has a parameters property which only includes the template parameters.

Happy to leave it as is. I don't feel strongly about it 😄 .

> Compile-time parameters don't seem to be a thing once the Stack instance has been generated.

Correct. They're used to _generate_ the template. Template parameters are _part of_ the template and evaluated at apply time by the CloudFormation service.

Reply:
👍 

I think I'll leave it as is. Differentiating between the two types of parameters here seems to be exposing internal implementation of the method. The output `Stack` instance names the template parameters, 'parameters', so I'll follow that convention.